external-docs/docs/guides/command-line/inspect-command.md

```diff
@@ -1,3 +1,939 @@
+co2         co2    5c1-2b  2016              waste-management                    not-applicable              waste-incineration incineration-chemical-waste                   other-emissions             other-emissions chemical-waste emissions-ar5-gwps  0.181
+co2         co2    5c1-2b  2016              waste-management                    not-applicable              waste-incineration incineration-clinical-waste                   other-emissions             other-emissions clinical-waste emissions-ar5-gwps  0.082
+co2         co2    5c1-2b  2017              waste-management                    not-applicable              waste-incineration incineration-chemical-waste                   other-emissions             other-emissions chemical-waste emissions-ar5-gwps  0.176
+co2         co2    5c1-2b  2017              waste-management                    not-applicable              waste-incineration incineration-clinical-waste                   other-emissions             other-emissions clinical-waste emissions-ar5-gwps  0.081
+co2         co2    5c1-2b  2018              waste-management                    not-applicable              waste-incineration incineration-chemical-waste                   other-emissions             other-emissions chemical-waste emissions-ar5-gwps  0.159
+co2         co2    5c1-2b  2018              waste-management                    not-applicable              waste-incineration incineration-clinical-waste                   other-emissions             other-emissions clinical-waste emissions-ar5-gwps  0.081
+co2         co2    5c1-2b  2019              waste-management                    not-applicable              waste-incineration incineration-chemical-waste                   other-emissions             other-emissions chemical-waste emissions-ar5-gwps  0.158
+co2         co2    5c1-2b  2019              waste-management                    not-applicable              waste-incineration incineration-clinical-waste                   other-emissions             other-emissions clinical-waste emissions-ar5-gwps  0.084
+co2         co2    5c1-2b  2020              waste-management                    not-applicable              waste-incineration incineration-chemical-waste                   other-emissions             other-emissions chemical-waste emissions-ar5-gwps  0.167
+co2         co2    5c1-2b  2020              waste-management                    not-applicable              waste-incineration incineration-clinical-waste                   other-emissions             other-emissions clinical-waste emissions-ar5-gwps  0.081
+        
+
+- The data cube has the following value counts:
+        - Value counts broken-down by measure and unit (of measure):
+           Measure   Unit  Count
+emissions-ar4-gwps MtCO2e  49765
+emissions-ar5-gwps MtCO2e  49765
+```
+
+#### Inspecting a multi-unit single-measure data cube
```

Comment:
Let's not do quite so many examples. One dataset & one code list should be enough. I'm debating whether that's too much info anyway. Perhaps these examples could have their own page and be linked to from the main one? 

That way users don't end up being overhwhelmed.